src/views/Project/index.jsx

```diff
@@ -117,8 +108,20 @@ export default class Project extends Component {
             Bugs & Issues
           </Typography>
         </header>
-        <TasksTable items={data} />
+        {error && <ErrorPanel error={error} />}
+        {loading && <Spinner />}
+        {!error && !loading && <TasksTable items={data} />}
       </Fragment>
     );
   }
 }
+
+const ProjectClient = props => (
+  <Query
+    query={gql`{${githubQuery(props.match.params.project)}}\n${Issues}`}
+    skip={projects[props.match.params.project].repositories === undefined}>
```

Comment:
Use a truthiness comparison instead of comparing to `undefined`.

```js
skip={!projects[props.match.params.project].repositories}
```

Reply:
noted